src/main/java/duke/Task.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.io.Serializable;
+
+/**
+ * Tasks are the tasks sent by the user to the Duke class
+ */
+public abstract class Task implements Serializable {
+    /**The name of the task.*/
+    private String taskName;
+    /**Status on whether the task is done or not.*/
+    private boolean taskStatus;
```

Comment:
Boolean variables should be named to sound like booleans.

Reply:
Yep, so perhaps this could be renamed as "isDone", or "isCompleted"

Reply:
Thank you for your feedback! I missed that out...